test/elixir_internal_certificate_web/features/home_page/search_keyword_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule ElixirInternalCertificateWeb.HomePage.SearchKeyword do
+  use ElixirInternalCertificateWeb.FeatureCase, async: true
+
+  feature("search box component", %{session: session},
+    do:
```

Comment:
Should be do/be block as per convention

Reply:
Updated in [28f8882](https://github.com/nvminhtue/elixir_internal_certificate/pull/38/commits/28f8882708ab0dc7e84e09e738b1c743527875eb)